client/src/pages/MainMenu.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "@emotion/styled";
+import Header from "../components/HeaderGoBack";
+import MenuAvailable from "../assets/menu-available.svg";
```

Comment:
You could name your imgSrc as following:
```suggestion
import MenuAvailableSrc from "../assets/menu-available.svg";
```

Reply:
Done ✅